modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -56,19 +53,50 @@ mirror:
     architectures:
       - "multi"
     channels:
-      - name: stable-4.13
-        minVersion: 4.13.4
-        maxVersion: 4.13.6
+      - name: stable-4.15
+        minVersion: 4.15.13
+----
+
+[discrete]
+[id="oc-mirror-image-set-examples-shortest-upgrade-path_{context}"]
+== Preparing for an OCP upgrade - including the shortest {product-title} update path
+
+The following ImageSetConfiguration file uses a local storage backend and includes all OpenShift Container Platform versions along with the shortest update path from the minimum version of 4.14.25 to the maximum version of 4.15.13.
+
+.Example ImageSetConfiguration file
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
+  local:
+    path: /home/user/metadata
+mirror:
+  platform:
+    channels:
+      - name: stable-4.15
+        minVersion: 4.14.25
+        maxVersion: 4.15.13
+        shortestPath: true
 ----
 
+you are setting it on, or include a different channel that has this version. 
```

Comment:
what does it mean by `you are setting it on` ?

Reply:
changed